sql/schema.sql

```diff
@@ -0,0 +1,125 @@
+CREATE TABLE tests (
+  id serial PRIMARY KEY,
+  name text NOT NULL UNIQUE,
+  run_frequency_mins INT NOT NULL,
+  method text NOT NULL,
+  url text NOT NULL,
+  headers JSONB,
+  payload JSONB,
+  query_params JSONB,
+  teardown text,
+  status text NOT NULL,
+  created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+  updated_at TIMESTAMP NOT NULL
```

Comment:
What do you think about removing the NOT NULL constraint on `updated_at`? That way we can leave null when a test is first created and populate on the first update